<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="accountsListView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>276</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuActions_2">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionLaunch_Browser"/>
    <addaction name="actionCopy_Password"/>
    <addaction name="actionCopy_Old_Password_2"/>
    <addaction name="separator"/>
    <addaction name="actionSynchornize"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuActions_2"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate"/>
   <addaction name="actionCopy_Password"/>
   <addaction name="actionLaunch_Browser"/>
   <addaction name="actionPreferences"/>
   <addaction name="actionSynchornize"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons8-plus-50.png</normaloff>:/icons8-plus-50.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="toolTip">
    <string>Create Account</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons8-edit-50.png</normaloff>:/icons8-edit-50.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons8-trash-can-50.png</normaloff>:/icons8-trash-can-50.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionCopy_Password">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons8-copy-to-clipboard-50.png</normaloff>:/icons8-copy-to-clipboard-50.png</iconset>
   </property>
   <property name="text">
    <string>Copy Password</string>
   </property>
  </action>
  <action name="actionSynchornize">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons8-download-from-cloud-50.png</normaloff>:/icons8-download-from-cloud-50.png</iconset>
   </property>
   <property name="text">
    <string>Synchornize</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons8-gear-50.png</normaloff>:/icons8-gear-50.png</iconset>
   </property>
   <property name="text">
    <string>Preferences..</string>
   </property>
  </action>
  <action name="actionCopy_Old_Password_2">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons8-copy-to-clipboard-50.png</normaloff>:/icons8-copy-to-clipboard-50.png</iconset>
   </property>
   <property name="text">
    <string>Copy Old Password</string>
   </property>
  </action>
  <action name="actionLaunch_Browser">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons8-globe-50.png</normaloff>:/icons8-globe-50.png</iconset>
   </property>
   <property name="text">
    <string>Launch Browser</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
